webserver/views/data.py

```diff
@@ -93,7 +93,7 @@ def summary(mbid):
     except db.exceptions.NoDataFoundException:
         summary_data = {}
 
-    if "highlevel" in summary_data:
+    if "highlevel" in summary_data and summary_data["highlevel"]:
```

Comment:
It can be an empty dictionary. Probably from high-level extractor that didn't work correctly.